### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by urmikgov**
- remove dangling comma
- add padding below documents table
- phone number placeholder standardization
- remove active days column from notifications

### CODE REVIEW

File: alcs-frontend/src/app/features/application/documents/documents.component.html [hunk 1]
```diff
@@ -25,14 +25,16 @@ <h3>Documents</h3>
...
-      <span *ngIf="element.visibilityFlags.includes('C')" matTooltip="Commissioner"
-        >C<span *ngIf="!hasBeenSetForDiscussion">*</span>,
-      </span>
-      <span *ngIf="element.visibilityFlags.includes('G')" matTooltip="L/FNG">G, </span>
-      <span *ngIf="element.visibilityFlags.includes('P')" matTooltip="Public"
-        >P<span *ngIf="!hasBeenReceived">*</span></span
-      >
+      <span *ngIf="element.visibilityFlags.includes('A')" matTooltip="Applicant">A</span>
+      <ng-container *ngIf="element.visibilityFlags.includes('C')">
+        <span matTooltip="Commissioner">, C<span *ngIf="!hasBeenSetForDiscussion">*</span></span>
```
Comment (trslater) [inline alcs-frontend/src/app/features/application/documents/documents.component.html:30]: What if `visibilityFlags` includes `'C'`, but not `'A'`?

File: alcs-frontend/src/app/features/home/assigned/assigned-table/assigned-table.component.ts [hunk 2]
```diff
@@ -22,11 +22,29 @@ export interface AssignedToMeFile {
 export class AssignedTableComponent {
   @Input() assignedFiles: AssignedToMeFile[] = [];
 
-  displayedColumns = ['highPriority', 'title', 'type', 'activeDays', 'stage'];
+  // displayedColumns = ['highPriority', 'title', 'type', 'activeDays', 'stage'];
```
Comment (trslater) [inline alcs-frontend/src/app/features/home/assigned/assigned-table/assigned-table.component.ts:25]: This should probably be deleted.

File: alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.html [hunk 3]
```diff
@@ -25,11 +25,17 @@ <h3>Documents</h3>
...
     <th mat-header-cell *matHeaderCellDef mat-sort-header sortActionDescription="Sort by visibility">Visibility</th>
     <td mat-cell *matCellDef="let element">
-      <span *ngIf="element.visibilityFlags.includes('A')" matTooltip="Applicant">A, </span>
-      <span *ngIf="element.visibilityFlags.includes('G')" matTooltip="L/FNG">G, </span>
-      <span *ngIf="element.visibilityFlags.includes('P')" matTooltip="Public"
-        >P<span *ngIf="!hasBeenReceived">*</span></span
-      >
+      <ng-container *ngIf="element.visibilityFlags.includes('A')">
+        <span matTooltip="Applicant">A</span>
+        <span *ngIf="element.visibilityFlags.includes('G') || element.visibilityFlags.includes('P')">, </span>
```
Comment (trslater) [inline alcs-frontend/src/app/features/notice-of-intent/documents/documents.component.html:30]: Here, you accounted for the issue above.

File: alcs-frontend/src/app/features/application/documents/documents.component.html (hunk 1 above)
Comment (urmikgov) [reply alcs-frontend/src/app/features/application/documents/documents.component.html:30]: will check this case

File: alcs-frontend/src/app/features/home/assigned/assigned-table/assigned-table.component.ts (hunk 2 above)
Comment (urmikgov) [reply alcs-frontend/src/app/features/home/assigned/assigned-table/assigned-table.component.ts]: removed

**Review by trslater — APPROVED** [review]
LGTM 🚀

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**